graphql_api/schema.py

```diff
@@ -1,8 +1,7 @@
-from ariadne import make_executable_schema, snake_case_fallback_resolvers
+from ariadne import make_executable_schema
 
 from .types import bindables, types
 
-# snake_case_fallbck_resolvers gives use a default resolver which convert automatically
-# the field name from camelCase to snake_case and try to get it from the object
-# see https://ariadnegraphql.org/docs/resolvers#fallback-resolvers
-schema = make_executable_schema(types, *bindables, snake_case_fallback_resolvers)
+# convert_names_case automatically converts the field name from camelCase
+# to snake_case. See: https://ariadnegraphql.org/docs/api-reference#optional-arguments-10
+schema = make_executable_schema(types, *bindables, convert_names_case=True)
```

Comment:
Nice inventorying, thanks for all the work!

Any potential cases of below? Could see them being gotchas-
* Anywhere we've appended numbers or special characters? I could see that getting resolved unexpectedly (e.g., myName0 could be `my_name_0` or `my_name0`) based on how the casing resolver wants to handle it.
* Any implicit case sensitivity handling we were getting "for free" in the old resolver that may be missing in the new? As in the old resolver was converting `MyName` --> `My_name` that now treated like `my_name` or something?

Based on the contents of this PR/ your sweep, it seems like No, but thought I'd put it out there!

Reply:
Great questions! The answer to the first is no special characters or numbers, at least none that I came across while doing this investigation.

The one "potential" key that I'm a little concerned with is "sessionid," but I think since it's fully lowercase we shouldn't see any unintended side effects from this change.

In terms of keys starting with a capital, my hope would be an ESLint rule would have caught that in Gazebo prior to it getting merged 😂 Capitals in the middle of a word where we didn't expect a "_" to occur would be problematic though, we can keep an eye on this in Sentry and see if anything pops up!

Reply:
Sounds good, thanks for the thorough checks!! Must have been tedious 😂